lib/Dialect/SV/Ops.cpp

```diff
@@ -113,6 +113,29 @@ static void printModportStructs(OpAsmPrinter &p, Operation *,
   p << ')';
 }
 
+/// Ensure that the symbol being instantiated exists and is an InterfaceOp.
+static LogicalResult verifyInterfaceInstanceOp(InterfaceInstanceOp &op) {
+  auto symtable = SymbolTable::getNearestSymbolTable(op);
+  if (!symtable) {
+    op.emitError("sv.interface.instance must exist within a region which has a "
```

Comment:
nit: I think you can `return op.emitError(...)` to make this a little more terse.